iSeg/SlicesHandler.cpp

```diff
@@ -3908,6 +3522,14 @@ void SlicesHandler::clear_overlay()
 		_overlay[i] = 0.0f;
 }
 
+void SlicesHandler::new_overlay()
+{
+	if (_overlay != nullptr)
+		free(_overlay);
```

Comment:
this does not look very C++... free and malloc...

Reply:
legacy code, but I cannot delete something that was malloc'd somewhere else.
maybe next refactoring I will replace the overlay by a std::vector<float>